desktop/cmp/input/Select.ts

```diff
@@ -435,11 +434,11 @@ class SelectInputModel extends HoistInputModel {
     // (Exception for a null value, which we will only accept if explicitly present in options.)
     override toInternal(external) {
         if (this.multiMode) {
-            if (external == null) external = []; // avoid [null]
+            if (external == this.emptyValue) external = []; // avoid [null]
```

Comment:
maybe we should be checking null OR emptyValue here?

Reply:
I'm not sure we want to be comparing to `this.emptyValue` in general here.

* We def. want to keep the null check, as we might have new prop set to `emptyValue: []` but still need to take incoming external value of `null` to `[]` for the commented reason (avoid [null] in the map below).
* I think of new `emptyValue` as only applying for _outbound_ values - i.e. the control has ended up empty through one course or another, so return this value instead.  Comparing an _inbound_ value here is something different - more like "always treat this value as empty".  I wonder if we want that here (and below on 441 for the single-value case).  Was this part of the change necessary to avoid some obviously wrong behavior?
* `==` comparison won't return true if an array involved anyway (unlikely to be reference to the _exact same_ empty array instance)